apps/agent/src/main/api.ts

```
@@ -0,0 +1,80 @@
+	startedAt: string,
+	activities: any,
+	timeLogId?: string,
+	organizationId: string,
+	tenantId: string,
+	organizationContactId?: string,
+	recordedAt: string
+}
+
+export class ApiService {
+	constructor() {
+	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)